lib/factories.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Bigben.Factories do
```

Comment:
You're welcome, and yeah I'm not sold on it either, but I'm trying it out as well.